src/js/redux/reducers/index.js

```diff
@@ -36,7 +37,8 @@ const appReducer = combineReducers({
     bulkDownload: bulkDownloadReducer,
     redirectModal: redirectModalReducer,
     stateProfile: stateReducer,
-    awardV2: awardV2Reducer
+    awardV2: awardV2Reducer,
+    searchSubAwardTable: searchSubAwardTableReducer
```

Comment:
If we do that then we need to move the filters and applied filters into one property called advanced search. I really do not think cleaning up our redux should be part of this ticket since it will blow up the scope. I am following the pattern of the way our current Redux is set up.